app/code/Magento/Backend/view/adminhtml/templates/widget/form/renderer/fieldset.phtml

```diff
@@ -51,12 +51,12 @@ if ($isField) {
             data-role="<?=  $block->escapeHtmlAttr($id) ?>-content">
 <?php endif; ?>
 
-    <?php if (!$element->getNoContainer()) : ?>
+    <?php if (!$element->getNoContainer()): ?>
         <fieldset class="<?=  $block->escapeHtmlAttr($cssClass) ?>" id="<?=  $block->escapeHtmlAttr($id) ?>">
-        <?php if (strlen($element->getBeforeElementHtml())) : ?>
+        <?php if (strlen(is_string($element->getBeforeElementHtml())) ? $element->getBeforeElementHtml() : ''): ?>
```

Comment:
I think the condition should look like proposed below
```suggestion
        <?php if (is_string($element->getBeforeElementHtml()) && strlen($element->getBeforeElementHtml()): ?>
```
@SilinMykola how do you think?

Reply:
@andrewbess Hi! Thanks for your advice, I made needed changes.